src/components/RestaurantDetails/components/Loader/Loader.js

```diff
@@ -0,0 +1,19 @@
+//Libraries
+import React from 'react';
```

Comment:
loader should be at common place since everywhere in the application you will use the same loader component

Reply:
As for now it is only used in restaurant details component so I choose to keep here. In next PR as it is used in both restaurant Details and menu component I moved to common components